SPECS/azurelinux-sysinfo/collect-sysinfo

```diff
@@ -220,10 +220,36 @@ def get_selinux_mode():
         ["getenforce"], capture_output=True, text=True, check=True
     ).stdout.strip()
 
+def get_system_manufacturer():
+    return subprocess.run(
+        ["cat", "/sys/devices/virtual/dmi/id/sys_vendor"], capture_output=True, text=True
+    ).stdout.strip()
+
+def get_host_type(sys_vendor):
+    if sys_vendor == "QEMU":
```

Comment:
Maybe you can compare this to a list of vendors instead, so it captures the different scenarios - ie: ['VMware', 'Microsoft Corporation', 'QEMU']